src/requirements/data/majors/math.ts

```diff
@@ -0,0 +1,396 @@
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          [
+            'CS 3220',
+            'CS 4110',
+            'CS 4160',
+            'CS 4210',
+            'CS 4220',
+            'CS 4620',
+            'CS 4670',
+            'CS 4700',
+            'CS 4740',
+            'CS 4744',
+            'CS 4775',
+            'CS 4780',
+            'CS 4786',
+            'CS 4787',
+            'CS 4810',
+            'CS 4812',
+            'CS 4814',
+            'CS 4820',
+            'CS 4830',
+            'CS 4850',
+            'CS 4852',
+            'CS 4860',
+          ],
+          [
+            'CS 3220',
+            'CS 4110',
+            'CS 4160',
+            'CS 4210',
+            'CS 4220',
+            'CS 4620',
+            'CS 4670',
+            'CS 4700',
+            'CS 4740',
+            'CS 4744',
+            'CS 4775',
+            'CS 4780',
+            'CS 4786',
+            'CS 4787',
+            'CS 4810',
+            'CS 4812',
+            'CS 4814',
+            'CS 4820',
+            'CS 4830',
+            'CS 4850',
+            'CS 4852',
+            'CS 4860',
+            'MATH 3***',
+            'MATH 4***',
+            'MATH 5***',
+            'MATH 6***',
+            'MATH 7***',
+          ]
+        ),
+        perSlotMinCount: [1, 3, 1],
+        slotNames: ['3000+ Math Course', 'Mathematical CS Courses', 'MATH or CS Course'],
+      },
+      Economics: {
+        description:
+          '5 courses from (i), (ii), and (iii): ' +
+          '(i) At least one MATH course numbered 3000 or above, ' +
+          '(ii) At least three ECON courses with significant mathematical content, ' +
+          '(iii) A course in ORIE with significant mathematical content dealing with material of interest in economics',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          [
+            'ECON 3130',
+            'ECON 3140',
+            'ECON 3810',
+            'ECON 3810',
+            'ECON 3825',
+            'ECON 4020',
+            'ECON 4110',
+            'ECON 4907',
+            'ECON 6090',
+            'ECON 6100',
+            'ECON 6130',
+            'ECON 6140',
+          ],
+          [
+            'ORIE 3300',
+            'ORIE 3310',
+            'ORIE 4350',
+            'ORIE 4600',
+            'ORIE 4740',
+            'ORIE 5600',
+            'ORIE 5610',
+            'ECON 3130',
+            'ECON 3140',
+            'ECON 3810',
+            'ECON 3810',
+            'ECON 3825',
+            'ECON 4020',
+            'ECON 4110',
+            'ECON 4907',
+            'ECON 6090',
+            'ECON 6100',
+            'ECON 6130',
+            'ECON 6140',
+            'MATH 3***',
+            'MATH 4***',
+            'MATH 5***',
+            'MATH 6***',
+            'MATH 7***',
+          ]
```

Comment:
Shouldn't this third slot just be the ORIE courses that are listed?

![Screen Shot 2021-04-17 at 10 11 34 AM](https://user-images.githubusercontent.com/54298311/115115989-4ac6fc00-9f65-11eb-98fb-51f16c75fcca.png)

Reply:
No, there isn't a minimum number of classes you need to take for the ORIE category. 

3/5 must be econ, 1/5 must be math, and the fifth class can be ORIE, MATH, or ECON. which is why all of them are listed.